sparse/numba_backend/_common.py

```diff
@@ -2136,6 +2152,19 @@ def astype(x, dtype, /, *, copy=True):
 
 @_support_numpy
 def squeeze(x, /, axis=None):
+    """Remove singleton dimensions from array.
+
+    Parameters
+    ----------
+    x : SparseArray
+    axis : int or tuple[int, ...], optional
+        The singleton axes to remove. By default all singleton axes are removed.
+
+    Returns
+    -------
+    SparseArray
```

Comment:
```suggestion
    output : SparseArray
```

Reply:
Done in https://github.com/pydata/sparse/pull/691/commits/1cf01c9a95c1095c48eb65bf1e39ba6e8ce9a3ab